alicja/config/proto/interface.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
```

Comment:
Nit: If the syntax is proto3, then we can't use the `required` or `optional` keywords in this file.